styleguide/components/callouts/callouts.stories.mdx

```diff
@@ -0,0 +1,94 @@
+import { Meta, Story, Preview, Source } from '@storybook/addon-docs/blocks';
+
+<Meta
+  title="Components/Callouts"
+  component={() => templateDefault}
+  decorators={[(Story) => `<div style="max-width: 800px">${Story()}</div>`]}
+/>
+
+# Callouts
+
+There are several different variants of the callout component:
+
+## Default
+
+import templateDefault from './_default.html.haml';
+import templateDefaultCode from '!!raw-loader!./_default.html.haml';
```

Comment:
`!!raw-loader!` is what allows us to show useful code snippets. Loads the mini Haml story partials as raw text.